modules/bastion/main.tf

```diff
@@ -100,41 +84,22 @@ module "ec2_bastion" {
   # Version 0.25.0 of this module did not assign an EIP, so we do it
   # in this component. Preserve backward compatibility by disabling it.
   assign_eip_address = false
+  zone_id            = try(length(var.vanity_domain), 0) > 0 ? data.aws_route53_zone.route53_zone[0].id : null
+  host_name          = var.custom_bastion_hostname
 
-  ebs_block_device_volume_size = 0
+  ebs_block_device_volume_size = var.ebs_block_device_volume_size
   ebs_delete_on_termination    = var.ebs_delete_on_termination
   user_data_base64             = join("", data.cloudinit_config.config.*.rendered)
   security_group_rules         = var.security_group_rules
   key_name                     = module.aws_key_pair.key_name
   instance_profile             = join("", aws_iam_instance_profile.default.*.name)
+  user_data                    = var.user_data
 
```

Comment:
If this is passed in, it will disable `user_data_base64`. Might be worth documenting this. I believe the upstream module variables do document this. Could be done in a follow up pr.

Reply:
Will account for this

Reply:
Resolved